index.js

```diff
@@ -354,3 +356,14 @@ form.addEventListener('click', (event) => {
     errorMsg.textContent = '';
   }
 });
+
+// code for data preservation in browser
+form.addEventListener('input', () => {
+  localStorage.setItem('text-input', username.value);
+  localStorage.setItem('email-input', mail.value);
+  localStorage.setItem('msg-input', textArea.value);
+});
```

Comment:
- [ ] The form data for all inputs should be stored in a single object, saved to a single local storage registry. Kindly do so
- [ ] In order to be saved to the local storage the data object is turned into a string using `JSON.stringify` or similar method. Kindly ensure this requirement is aso met